cmd/empire/main.go

```diff
@@ -66,6 +69,12 @@ var Commands = []cli.Command{
 				Usage: "Whether to run the migrations at startup or not",
 			},
 			cli.StringFlag{
+				Name:   FlagEventsBackend,
+				Value:  "",
+				Usage:  "The backend implementation to use to send event notifactions",
```

Comment:
Not out of the box, but I think we could implement our own StringFlag based on [GenericFlag](https://github.com/codegangsta/cli/blob/master/flag.go#L59-L70). I'll take a stab at that.